### PR TITLE
Responsive images

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- Responsive images are here!
- This change allows you to specify a range of images for display at different widths using the `srcset` attribute.

#### What tests does this PR have?
None yet - this will not be merged

#### How can this be tested?
- Clone the UI Toolkit
- Run `grunt build`
- Run `grunt docs`
- When the docs have loaded up, go to the image component and view the responsive section (a cute panda)
- Shrink your browser window to a small size (smaller... smaller still) and hard refresh the page. You should see a green panda image with 1000w written on it.
- As you expand the window this image will change to blue (1500w) and red (2000w). This is swapping out images in a responsive manner.
- If you now shrink your browser back down to mobile, it will stay red (2000w) as that asset has already been downloaded.

if you need me to demo this, please ask :)

#### Screenshots / Screencast
screen shot 2015-09-22 at 14 16 05
screen shot 2015-09-22 at 14 15 10
screen shot 2015-09-22 at 14 15 22

#### What gif best describes how you feel about this work?
Panda

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

### CODE REVIEW

**Comment by cameronviner** [comment]
Taking DEV

**Comment by cameronviner** [comment]
Looks awesome :+1:

**Comment by jodiedoubleday** [comment]
:)

File: docs/src/Components.jsx [hunk 1]
```diff
@@ -69,8 +69,11 @@ var Components = React.createClass({
 
           <article id="image">
             <h3>Image</h3>
-            <p>A standard Image Component.</p>
+            <p>A standard Image Component which is set at 100% width by default to ensure the image works resposively.</p>
```
Comment (hxjamesmcneil) [inline docs/src/Components.jsx]: spelling `responsively`

**Comment by hxjamesmcneil** [comment]
Taking SE/Guru

File: src/components/image/__tests__/image-test.js [hunk 2]
```diff
@@ -86,7 +90,13 @@ describe('ImageComponent', function() {
       assert.ok(handleClick.calledOnce);
     });
 
-  });
+    it('should render multiple responsive images', function() {
+      assert.equal(imageDOMNode.getAttribute('srcset'), '2000w.jpg 2000w, 1500w.jpg 1500w, 1000w.jpg 1000w, 500w.jpg 500w');
+    });
 
+    it('should render a sepcified start image size', function() {
```
Comment (hxjamesmcneil) [inline src/components/image/__tests__/image-test.js]: spelling `specified`

**Pull request merged**